datasets/de/bundesrat/crawler.py

```diff
@@ -0,0 +1,86 @@
+import re
+from normality import squash_spaces
+from lxml.html import HtmlElement
+
+from zavod import Context, helpers as h
+from zavod.stateful.positions import categorise
+
+START_DATE_REGEX = re.compile(
+    r"(?:Stellvertretendes )?Mitglied des Bundesrates seit (\d{2}\.\d{2}\.\d{4})|,?\s*[Ss]eit (\d{2}\.\d{2}\.\d{4})\s*(?:Stellvertretendes\s+)?Mitglied des Bundesrates"
```

Comment:
I trust you it works, but what does this match? Does it do "Mitglied des Bundesrates seit [date1], seit [date2] Stellvertretendes..."

In that case, don't we only care about date1?

Could you add some examples that this matches in a comment? And maybe mention somewhere (metadata? crawl_item?) that we're ignoring deputy designations?